recipes/pdftopng/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set version = "0.2.3" %}
+
+package:
+  name: pdftopng
+  version: {{ version }}
+
+source:
+  - url: https://github.com/vinayak-mehta/pdftopng/archive/v{{ version }}.tar.gz
+    sha256: 2b0cda9b7e151d67a182bb753138309c3a6b035939577c5527a34a2c1ef96b64
+    patches:
+      - 0001-fix-setup.patch
+      - setOverprintPreview.patch
+  - url: https://github.com/freedesktop/poppler/raw/fcdff7bb19e2ac0fab6505f17e0c18c8faa86323/utils/numberofcharacters.h
+    sha256: 285258e387080c775e4f6f5bca652e9e707c3beac83c6625d3b840c4e13c6440
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - pybind11                               # [build_platform != target_platform]
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - setuptools
+    - cython
+    - pybind11
+    - poppler
+    - click
+    - pip
+  run:
+    - python
+    - poppler
+    - click
```

Comment:
Is there a console entry point for this package? You need to register entry points with conda. Also please add a call to the entry point in the tests section.

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points

Reply:
Hmm, is this really needed? The entry point is already specified in https://github.com/vinayak-mehta/pdftopng/blob/633c594f24e77f00f00f0675b49daea2c71e279b/setup.py#L171 and the package produces the appropriate scripts/binaries. Do we really need to specify it in `meta.yaml` also?

Reply:
I see https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=entry_point which suggests that at least a redundant entry point in `meta.yaml` won't hurt anything, so I've gone ahead and added it as requested.

Reply:
They are needed (especially for `noarch: python` packages)